pom.xml

```diff
@@ -116,15 +116,21 @@
 
         <dependency>
             <groupId>org.scalatest</groupId>
-            <artifactId>scalatest_2.11</artifactId>
-            <version>3.1.0</version>
+            <artifactId>scalatest_2.12</artifactId>
+            <version>3.0.5</version>
             <scope>test</scope>
         </dependency>
 
         <dependency>
             <groupId>com.github.scopt</groupId>
-            <artifactId>scopt_2.11</artifactId>
-            <version>3.7.1</version>
+            <artifactId>scopt_2.12</artifactId>
+            <version>3.5.0</version>
```

Comment:
Same - let's keep the version originally used (3.7.1)

Reply:
Noted. will be done in next commit.